app/ts/tests/ethSimulate.ts

```diff
@@ -0,0 +1,60 @@
+import { EthSimulateV1Params } from './ethSimulate-types.js'
+import * as funtypes from 'funtypes'
+import { serialize } from './wire-types.js'
+
+type JsonRpcSuccessResponse = funtypes.Static<typeof JsonRpcSuccessResponse>
+const JsonRpcSuccessResponse = funtypes.ReadonlyObject({
```

Comment:
```suggestion
export type JsonRpcSuccessResponse = funtypes.Static<typeof JsonRpcSuccessResponse>
export const JsonRpcSuccessResponse = funtypes.ReadonlyObject({
```

Reply:
Mostly for consistency.  For example, you export `EthereumJsonRpcRequest`.  In general, I like having all of my wire types exported so they are available where needed.  If you want to keep them unexported and are OK with the inconsistency then I won't block the PR on that.

Reply:
The point is to have things exported that are used used externally, these are not used externally